packages/@aws-cdk/aws-redshift/lib/cluster.ts

```diff
@@ -365,6 +379,11 @@ abstract class ClusterBase extends Resource implements ICluster {
    */
   public abstract readonly connections: ec2.Connections;
 
+  /**
+   * Indicates if the parameter group can be updated.
+   */
+  protected abstract autoCreateParameterGroup: boolean;
```

Comment:
docstring is terribly wrong here. The intent of this is to flag whether or not `addToParameterGroup` should create a parameter group. This would be the case on a `Cluster` that didn't have a group associated with it on instantiation.